database.json

```diff
@@ -12,5 +12,92 @@
+        "ICMLink": "https://portal.microsofticm.com/imp/v3/incidents/details/366336692/home",
+        "Description": "A sev 3 alert is fired for a 4xx or 5xx error in one of our endpoints.  Get to the root cause of the error and determine whether action must be taken, and how to keep other customers from experiencing the same issue in the future.",
+        "Questions": [
+            {
+                "Question:":"How many instances of this error have occurred within 20 minutes of the alert?",
+                "Answer": "8",
+                "Hints": [
+                    "This information can be found in the logs",
+                    "Open the relevant logs using a link from the ICM",
+                    "You might not see the relevent logs if too many results are returned",
+                    "Use the Filtering Conditions to scope down the search so you can see all the relevent results",
+                    "Make sure you refresh, sometimes not all the logs will be shown and a refresh should fix this",
+                    "Two logs should be present for each failed request"
+                ]
+            },
+            {
+                "Question:":"What is the correlation Id for one of the affected requests?",
+                "Answer": [
+                    "e9995d417f66484783bf895f8c045f66",
+                    "ff11cf2250034a76a4d6122a21d2f0b9",
+                    "cc259609a0094ae5ae6d4976277e4c36",
+                    "1b820e370c0e414cb37c29f7a0094f0c",
+                    "62ac23ba85f94b44b6ef9ba678c6ec45",
+                    "3c9aed928b604bb59cc4f5c5f3450f7a",
+                    "9953bb11181f4bef9f21ee3a3f51f5d9",
+                    "85fa87bd8b364723828ae7dc15cdb9b1"
+                ],
+                "Hints": [
+                    "This can be found in the logs, or one of the correlation Ids can be found in the ICM itself",
+                    "Look for a Troubleshooting information (update) message in the discussion section",
+                    "Use the same log search as before"
+                ]
+            },
+            {
+                "Question:":"What is the correlationVector for the affected request?",
+                "Answer": [
+                    "m7TcM/K6QESTUAP/jbvF+w",
+                    "Nu/I3T46IUG8yOGa65KS7A",
+                    "noy8vmqf9EaEDKyM2nIS0w",
+                    "sTZd9h0v1EqYY289qH5dbw",
+                    "ToKjJdZ1C0G+DBCh0pawEg",
+                    "EJbMMJJW0EyB8rhqRFIUiQ",
+                    "lkGUj5oi+0Ct4AZRTRirQQ",
+                    "KXQxfJZAAEmrRn6h0OHLpg"
+                ],
+                "Hints": [
+                    "This information can be found in the logs",
+                    "Use the same log search as before"
+                ]
+            },
+            {
+                "Question:":"What was the underlying error message for the failed request?",
+                "Answer": "The Resource Provider returned '499' status code.",
```

Comment:
when I am doing interpolation on the string, it returns an error, just for a happy path can you remove the '' around 499.